doc/projects_list.txt

```diff
@@ -0,0 +1,29 @@
+* :ref:`Sphinx-Gallery <examples-index>`
```

Comment:
Should this file extension be `.rst` since we are using rst syntax?

Reply:
I just used the same file name (I mean the base name of the file, without the extension) `projects_list` for the include file and the corresponding docs file, that's why I used `.txt` for the former. Other solutions would be

1. rename `projects.rst` to `who_uses.rst` and then `projects_list.txt` to `projects_list.rst` to include the latter in `README.rst` and `who_uses.rst`  or   
2. put a label in the existing `projects.rst` where the list starts and include `start-after` this label

I'm open for both options - your choice.

Reply:
2 seems cleaner. Thanks!

Reply:
done